### PR TITLE
[ASEO-41] WP and FE: Show blocks from WP

### PR DESCRIPTION
**Pull request by Idan-Garay**
- added await to line 39 in dbForFrontend to fix issue on global.mongo.client.db is not a function
- fetch review article data from WP and parse it into react children
- added wpStyle.css for styling the parsed React Children from WP

### CODE REVIEW

File: app/_components/Content/ArticleHeader/index.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import CalendarIcon from "assets/img/Calendar";
 import PeopleIcon from "assets/img/People";
 import { Author, BreadcrumbItem } from "app/best-games/types";
 import ArticleHeaderDetailItem from "app/_components/Content/ArticleHeaderDetailItem";
+import React from "react";
```
Comment (idiglove) [inline app/_components/Content/ArticleHeader/index.tsx]: do we need this?

File: app/_components/Tournaments/styled.ts [hunk 2]
```diff
@@ -139,7 +139,7 @@ export const SlantedDiv = styled.div`
   height: 52px;
   width: 49.134rem;
   overflow: hidden;
-  clip-path: polygon(0 0, 81% 0, 100% 100%, 0 100%);
+  clippath: polygon(0 0, 81% 0, 100% 100%, 0 100%);
```
Comment (idiglove) [inline app/_components/Tournaments/styled.ts]: is this a needed change?

File: assets/img/ALPACA.svg
Comment (idiglove) [inline assets/img/ALPACA.svg:1]: what happened to this file?

File: assets/img/ChevronRight.js [hunk 3]
```diff
@@ -19,8 +19,8 @@ const ChevronRight = ({ ...props }) => (
     </mask>
     <g mask="url(#mask0_867_3818)">
       <path
-        fill-rule="evenodd"
-        clip-rule="evenodd"
+        fillRule="evenodd"
```
Comment (idiglove) [inline assets/img/ChevronRight.js:22]: 👌👌

File: middlewares/dbForFrontend.js [hunk 4]
```diff
@@ -34,8 +35,8 @@ export default async function database(req, _, next) {
 
   let mongodbName =
     process.env.ENV === "test" ? process.env.DB_NAME_TEST : process.env.DB_NAME;
-  let db = global.mongo.client.db(mongodbName);
 
+  let db = (await global.mongo.client).db(mongodbName);
```
Comment (idiglove) [inline middlewares/dbForFrontend.js:39]: thanks!!

File: app/_components/Content/ArticleHeader/index.tsx (hunk 1 above)
Comment (Idan-Garay) [reply app/_components/Content/ArticleHeader/index.tsx]: This is needed, I'm using React.fragment in this file

File: app/_components/Tournaments/styled.ts (hunk 2 above)
Comment (Idan-Garay) [reply app/_components/Tournaments/styled.ts]: Nope

**Review by idiglove — APPROVED** [review]
Just make the changes like I said in Discord. Other than that LGTM! 🏄🏽

**Pull request merged**